File: app/src/main/java/org/mifos/mobilebanking/models/accounts/LoanAccountsListResponse.java
package org.mifos.mobilebanking.models.accounts;

import org.mifos.mobilebanking.models.accounts.loan.LoanAccount;

import java.util.ArrayList;
import java.util.List;

/**
 * @author Vishwajeet
 * @since 13/08/16
 */
public class LoanAccountsListResponse {
    private List<LoanAccount> loanAccounts = new ArrayList<>();

    public List<LoanAccount> getLoanAccounts() {
        return loanAccounts;
    }

    public void setAccounts(List<LoanAccount> loanAccounts) {
        this.loanAccounts = loanAccounts;
    }
}

File: app/src/main/java/org/mifos/mobilebanking/models/accounts/LoanAccountsListResponse.kt
package org.mifos.mobilebanking.models.accounts

import org.mifos.mobilebanking.models.accounts.loan.LoanAccount

import java.util.ArrayList

/**
 * @author Vishwajeet
 * @since 13/08/16
 */
data class LoanAccountsListResponse (
        var loanAccounts: List<LoanAccount> = ArrayList())
